schematic/schemas/commands.py

```diff
@@ -105,22 +115,38 @@ def convert(schema, data_model_labels, output_jsonld):
                 for w in war:
                     logger.warning(w)
 
+    if output_jsonld is None:
```

Comment:
These need to be updated to look for `.pickle` as well. For now if you put a `.pickle` file extension, for `output_jsonld` it will not be removed, so the resultant file will have two `.pickle` extensions.